hazelcast/src/main/java/com/hazelcast/internal/management/operation/ChangeWanStateOperation.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management.operation;
+
+import com.hazelcast.nio.ObjectDataInput;
+import com.hazelcast.nio.ObjectDataOutput;
+import com.hazelcast.spi.Operation;
+
+import java.io.IOException;
+
+/**
+ * Enables/Disable publishing events to target cluster in WAN Replication {@link com.hazelcast.wan.WanReplicationService}
+ * on a node. This operation does not block adding new events to event queue.
+ */
+public class ChangeWanStateOperation extends Operation {
+
+    private boolean start;
+
+    public ChangeWanStateOperation() {
+    }
+
+    public ChangeWanStateOperation(boolean start) {
+        this.start = start;
+    }
+
+    @Override
+    public void beforeRun() throws Exception {
+
+    }
+
+    @Override
+    public void run() throws Exception {
+        /*TODO*/
+    }
+
+    @Override
+    public void afterRun() throws Exception {
+
+    }
+
+    @Override
+    public boolean returnsResponse() {
+        return true;
+    }
+
+    @Override
+    public Object getResponse() {
+        return null;
+    }
+
+    @Override
+    protected void writeInternal(ObjectDataOutput out) throws IOException {
+
+    }
+
+    @Override
+    protected void readInternal(ObjectDataInput in) throws IOException {
+
+    }
+}
```

Comment:
@muratayan , Can you please extend this class from `AbstractOperation`?
We'll only need to implement `run` method, all others won't be needed.